.changeset/forty-mangos-judge.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-tokens': minor
+---
+
+Add motion prefix for motion tokens
```

Comment:
Is this a major since we are changing the token names?

Reply:
The intention is for this to not be a breaking change as we are adding new values not removing any. The breaking change will be removing the values.